src/resolver/Query/random.js

```diff
@@ -0,0 +1,18 @@
+import applyFilters from './applyFilters';
+
+const getRandomNumber = (min, max) =>
+    Math.floor(Math.random() * (max - min + 1)) + min;
+
+export default (entityData = []) => (_, { count = 3, filter = {} }) => {
+    let items = [...entityData];
+
+    items = applyFilters(items, filter);
+    // cut down to count items
+    let returnCount = Math.min(count, items.length);
+    let randomItems = [];
+    for (let i = 0; i < returnCount; i++) {
+        randomItems.push(items[getRandomNumber(0, returnCount - 1)]);
+    }
```

Comment:
This is likely wrong, see: [here](https://art-genome-project-24jvbvihi.now.sh/?query=%7B%0A%20%20randomGenes(count%3A5)%20%7B%0A%20%20%20%20name%0A%20%20%20%20family%0A%20%20%20%20description%0A%20%20%7D%0A%7D) (there are 1000+ genes, but it only cycles through a few of them)

Reply:
Fixed in 8a76ff71954da08ab4c2b8495a53e8f687ce5e2c